spec/unit/static_html_spec.rb

```diff
@@ -44,5 +41,21 @@
     )
   }
 
-  
+  describe "#initialize" do
+    it "initializes without error" do
+      expect { StaticHtml.new(description) }.not_to raise_error
+    end
+  end
+
+  describe "#write" do
+    it "renders an HTML report" do
+      static_html = StaticHtml.new(description)
+      Dir.mktmpdir do |tmp_dir|
+        static_html.write(tmp_dir)
+        index_file = File.join(tmp_dir, "index.html")
+        expect(File.readable?(index_file)).to be_truthy
+        expect(File.read(index_file)).to include "<html", "openSUSE"
```

Comment:
Hound doesn't cover it, but according to our [style guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md) you should use parentheses for all method calls (with exceptions for statement like calls like `puts foo` etc.)